routes/users.js

```diff
@@ -106,7 +106,7 @@ router.post(
 
 router.post(
   '/block-anon-user-from-chat',
-  auth.isAuth,
+  auth.isAuthV2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the authentication middleware has been updated from `isAuth` to `isAuthV2`. Make sure that `isAuthV2` is thoroughly tested and handles all edge cases that `isAuth` did. Also, ensure that the clients of this API are ready to handle any changes in the response or behavior due to this update.
```diff
-  auth.isAuth,
+  auth.isAuthV2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->